### DISCUSSION TITLE
Images in rich editor are stored as base64 string

### DISCUSSION THREAD

**Discussion by JustDenP** [pocketbase/pocketbase#4481] [Q&A] [answered]
Hello, I want to make a simple blog, I made a posts collection with 'content' field as text with rich editor support. I placed some text and image inside, but when I fetch the posts, I noticed that my page loads way to slow, and then I found that these images returned not as urls, but as a huge base64 string inside the content filed. Im I missing something? How to avoid that and get only urls? Thank you

![CleanShot 2024-03-03 at 20 31 30@2x](https://github.com/pocketbase/pocketbase/assets/19192519/da51f3ba-bd9e-44ed-865f-f84745302780)
![CleanShot 2024-03-03 at 20 32 59@2x](https://github.com/pocketbase/pocketbase/assets/19192519/a5447b5a-d1cd-454f-83a4-1edf5a2ff4f6)

**Comment by ganigeorgiev** [comment, marked as answer]
This is expected if you've selected the "Inline" image option from the editor.

Chose the "From collection" option to link uploaded files from collections with image file fields.

**Reply by ganigeorgiev** [reply]
If you don't have collections with `file` fields, note that you can also create a "Media" base collection with a single `file` field to store all your uploaded files and then use the "From collection" option to link images from it.

**Reply by JustDenP** [reply]
Thank you!
